Xamarin.Forms.Core/Cells/Cell.cs

```diff
@@ -12,6 +12,7 @@ public abstract class Cell : Element, ICellController, IElementConfiguration<Cel
 	{
 		public const int DefaultCellHeight = 40;
 		public static readonly BindableProperty IsEnabledProperty = BindableProperty.Create("IsEnabled", typeof(bool), typeof(Cell), true, propertyChanged: OnIsEnabledPropertyChanged);
+		public static readonly BindableProperty BackgroundColorProperty = BindableProperty.Create(nameof(BackgroundColor), typeof(Color), typeof(Cell), Color.Default, propertyChanged: OnBackgroundColorPropertyChanged);
```

Comment:
are cells stylable with CSS ? if so, this should either share the property with VE, or you should add a StyleProperty in AssemblyInfo.cs

Reply:
AFAIK no css for cells